compiler/rustc_passes/src/weak_lang_items.rs

```diff
@@ -70,7 +70,19 @@ fn verify<'tcx>(tcx: TyCtxt<'tcx>, items: &lang_items::LanguageItems) {
                     tcx.sess.note_without_error("Use `#![feature(default_alloc_error_handler)]` for a default error handler");
```

Comment:
```suggestion
                    tcx.sess.note_without_error("use `#![feature(default_alloc_error_handler)]` for a default error handler");
```

pre-existing: our diagnostics are typically always lowercase, could you add a commit changing this?